src/components/ButtonWithLoading/Loader/index.js

```diff
@@ -2,7 +2,7 @@ import React from "react";
 import PropTypes from "prop-types";
 
 import { Dot, LoaderWrapper, ContentWrapper, DotsWrapper } from "./styles";
-import { BUTTON_VARIANTS } from "../../constants";
+import { BUTTON_VARIANTS, STANDARD } from "../../constants";
 
 const Loader = ({ children, isLoading, variant }) => (
   <LoaderWrapper>
```

Comment:
`LoaderWrapper` and `DotsWrapper` are adding span tags to the button. I think if the consumer wants to use any other tags such as `svg` or `div` tags within the button, it is not a good practice to pass other elements into the span tag. Or we can leave for the user to set the CSS for the inner elements. What do u think @nyonchev and @io-rozhkov ?